ui/jquery.ui.droppable.js

```diff
@@ -275,6 +282,16 @@ $.ui.ddmanager = {
 		}
 
 	},
+	addDroppable: function ( scope, droppable ) {
+		$.ui.ddmanager.droppables[ scope ] = $.ui.ddmanager.droppables[ scope ] || [];
+		// TODO: Binary search would be faster
```

Comment:
Actually, just drop this. If it ever becomes a problem, someone will file a bug and we can deal with it then. Otherwise, this todo will just sit forever.